.github/workflows/release.yml

```diff
@@ -89,12 +129,15 @@ jobs:
         id: deployment
         uses: actions/deploy-pages@v4
 
+      - name: Release Please GitHub Action
```

Comment:
just for information, will this create a tag and GH release or just a tag?
And will the tag and/or release be prefixed with a `v`?

Reply:
Yes, it will create a tag and GH release and both are prefixed with v. 

**Example :**

![2024-08-12_11-51](https://github.com/user-attachments/assets/b98e8f45-6500-44a1-8627-e742b2580b93)
![2024-08-12_11-50](https://github.com/user-attachments/assets/8e7c76ed-a6c1-4be9-a0e6-3157cfecd72a)